test/org/sagebionetworks/bridge/services/SubpopulationServiceTest.java

```diff
@@ -157,6 +161,7 @@ public void createSubpopulation() {
         verify(subpopDao).createSubpopulation(subpop);
         verify(studyConsentService).addConsent(eq(result.getGuid()), any());
         verify(studyConsentService).publishConsent(study, result, CONSENT_CREATED_ON);
+        verify(substudyService).getSubstudyIds(TEST_STUDY);
```

Comment:
Should this be added to updateSubpopulation()?

Reply:
Yes, added